XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
+The Exchange Union project is powered by [XUD](https://github.com/ExchangeUnion/xud/), short for **Exchange Union Daemon**, the  open source node software which is operated by exchanges (or traders) and comprises the network. This document describes the inner workings of XUD from a product & technical perspective and summarizes technical requirements. For an overview of idea, use case & benefits read the [White Paper](https://www.exchangeunion.com/static/files/ExchangeUnion-WhitePaper_en.pdf), for hands-on guides check our [Wiki](https://github.com/ExchangeUnion/xud/wiki) and for code-level documentation our [TypeDoc](https://exchangeunion.github.io/xud-typedoc/). Chat with us on [Gitter](https://gitter.im/exchangeunion/Lobby) and follow major event & development milestones on our [blog](https://blog.exchangeunion.com/).
+
+XUD's code base and documentation is available under the [AGPL-3.0](https://github.com/ExchangeUnion/xud/blob/master/LICENSE) open-source license. The objective is to build a open-source developer community, while additionally employing full-time developers under the Exchange Union brand. Development is funded through an initial private sale of 2% of Exchange Union's XUC token, which is since in circulation. No ICO or further fundraising is planned.
 
 
 ## 2. Product Description
-Exchange Union can be described as a *decentralized meta-exchange*, because it targets today's centralized exchanges as "users". This direction was chosen because the majority of users use and will use centralized exchanges for convenience and thus the liquidity is with centralized exchanges. Furthermore, executing and maintaining Exchange Union’s technology, XUD, short for **Exchange Union Daemon** is not expected to be suitable for personal or widespread usage in the near future. However, it is planned to further develop and simplify the technology and make it suitable for individual usage in the long-term. 
+From a business perspective, Exchange Union can be described as a *decentralized meta-exchange*: target users are existing centralized exchanges, in the following 'exchanges' for short.
 
-Simply put, Exchange Union’s vision is to offer individuals a choice to either:
+From a technical perspective, XUD spins up an *instant [DEX](https://en.wikipedia.org/wiki/Decentralized_exchange)* which can be openly accessed and used by everyone. However, due to its technical complexity, it is not expected that XUD will see a widespread adoption by individuals in the short-term.
```

Comment:
I don't think "from a technical perspective" is necessary. and you can just say "widespread adoption" instead of "a widespread adoption" in the second sentence.

Reply:
I just wanted to differentiate the two views: business perspective: connecting exchanges, tech perspective: instant DEX.

> widespread adoption

Fixed